battenberg/tmp_worktree.py

```diff
@@ -0,0 +1,60 @@
+import os
+import shutil
+import tempfile
+from types import TracebackType
+from typing import Optional, Type
+
+from pygit2 import Repository, Worktree
+from battenberg.errors import (
+    RepositoryEmptyException,
+    WorktreeConflictException,
+    WorktreeException
+)
+
+
+class TemporaryWorktree:
```

Comment:
Nit. do you want to rename the file to `temporary_worktree.py`?

Reply:
is this jut copy paste from the cut above?

Reply:
Yep I just moved it into a separate module to make it easier to test it out. No changes. I can rename the module to `temporary_worktree.py`.